src/T2DMap.cpp

```diff
@@ -672,13 +672,503 @@ bool T2DMap::sizeFontToFitTextInRect( QFont & font, const QRectF & boundaryRect,
     return true;
 }
 
+// This has been refactored to a separate function out of the paintEven() code
+// because we need to use it in two places - one for every room that is not the
+// player's room and then, AFTER all those have been drawn, once for the
+// player's room if it is visible. This is so it is drawn LAST (and any effects,
+// or extra markings for it do not get overwritten by the drawing of the other
+// rooms)...
```

Comment:
I don't think this comment is needed. We don't need the history of the code as a comment (we can look at git history for that) and we can tell the function is used in two places by looking at usages. Long comments don't really help in readability, plus it's only you who'd be interested in keeping it up to date...

Reply:
Well, not every usage of the source code is going to be from a git repository, but I take your point - how about if I reduced it to a one-liner along the lines of:
```
// We need to ensure the player's room is drawn last so that any oversized parts of it don't get overwritten by other rooms....
```

Reply:
Perfect, and that should be at the point where this function is called for that reason - not at the function definition (doesn't make sense there)